src/routes/Press/Home/PressHome.js

```diff
@@ -40,7 +41,9 @@ const PressHome = () => {
                 />
               ) : (
                 <div>
-                  <EmptyArticlesMessage type='press' />
+                  <EmptyMessage 
+                    entityName='Press'
+                    createCopy='Create a new Press' />
```

Comment:
can we make the message 'Create Press' :)

Reply:
changed and pushed.